pkg/testutil/env/env.go

```diff
@@ -39,7 +38,17 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/envtest"
 )
 
+// testBinaries is used to set KUBEBUILDER_ASSETS variable to ensure things like etcd are available to all test
+// environments. This path is also added to the PATH.
```

Comment:
just curious - why do we need to both set `KUBEBUILDER_ASSETS` and also add to `PATH`?

Reply:
When our test targets specify data resources like `//hack/bin:etcd`, it's typically for being able call `<thatpath>/etcd`. However, some of our packages require that the values be on the path, like _pkg/security_ for example. If this seems odd, we can separate the two and make two calls in `Env.Start`.

Reply:
it does seem odd and it might be good to separate the two (if only to have a better understanding of what all the dependencies are), but we can do that later

Reply:
Cool, I'll address it now just to avoid another TODO. Should be easy enough.